app/src/main/res/layout/modal_choose_pickup_address.xml

```diff
@@ -0,0 +1,192 @@
+            android:text="COMMON ADDRESSES"
+            android:textStyle="bold"
+            android:textColor="@color/colorTextIcons"
+            android:padding="8dp"
+            android:layout_marginBottom="8dp"/>
+
+        <LinearLayout
+            android:id="@+id/commonAddressesList"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/receiverHeaderEditText"
+            android:orientation="vertical"
+            android:layout_marginBottom="0dp">
+
+
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="Rua Sen. Fraga 1001 - Manguinhos"
+                android:textColor="@color/colorDivider"
+                android:textColorHint="@color/colorDivider"
+                android:background="@color/transparentWhite05"
+
+                android:textSize="16dp"
+                android:padding="8dp"
+                android:layout_marginBottom="8dp"/>
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="Av. Gomes Freire 290 - Centro"
+                android:textColor="@color/colorDivider"
+                android:textColorHint="@color/colorDivider"
+                android:background="@color/transparentWhite05"
+
+                android:textSize="16dp"
+                android:padding="8dp"
+                android:layout_marginBottom="8dp"/>
+
+
+        </LinearLayout>
+
+    </RelativeLayout>
+
+
+    <RelativeLayout
+        android:id="@+id/suggestedAddressesContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/commonAddressesContainer"
+        android:background="@color/transparentWhite05"
+        android:padding="8dp"
+        android:layout_marginBottom="20dp">
+
+        <TextView
+            android:id="@+id/suggestedAddressesHeaderEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center"
+            android:text="SUGGESTED ADDRESSES"
+            android:textStyle="bold"
+            android:textColor="@color/colorTextIcons"
+            android:padding="8dp"
+            android:layout_marginBottom="8dp"/>
+
+        <LinearLayout
+            android:id="@+id/suggestedAddressesList"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/suggestedAddressesHeaderEditText"
+            android:orientation="vertical"
+            android:layout_marginBottom="0dp">
+
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="Av. Gomes Freire 290 - Centro"
```

Comment:
This should be in the String file with the name: modalPickupAddress_txtView_hintSuggestedAddresses1.